### PR TITLE
DDDP 609

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "Offers",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "baseBundleVersion": "0.57.5",
   "private": true,
   "scripts": {
-    "start": "pushd mobile && npm start && popd && pushd web/admin && npm start && popd"
+    "start": "pushd mobile && npm run ios && popd && pushd web/admin && npm start && popd"
```
Comment (AdamLiechty) [inline package.json]: Why this change?

File: mobile/src/Offers.js [hunk 2]
```diff
@@ -32,6 +31,15 @@ export default class Offers extends Component {
     })
   }
 
+  componentDidUpdate(nextProps) {
+    if (nextProps !== this.props) {
```
Comment (AdamLiechty) [inline mobile/src/Offers.js]: Only run this if `props.image` changes?

**Review by AdamLiechty — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (edwardstamps) [reply package.json]: Not entirely sure how this change occurred after running upgrade-rn

Comment (AdamLiechty) [reply package.json]: 👍 looks like this had undone what upgrade-rn had done.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Offers",
-  "version": "1.0.4",
+  "version": "0.1.6",
```
Comment (AdamLiechty) [inline package.json]: Please be sure to view diff of changes before each commit to avoid committing this

**Review by AdamLiechty — CHANGES_REQUESTED** [review]
